test/metabase/api/dashboard_test.clj

```diff
@@ -2849,51 +2851,53 @@
                 (is (partial= {:message "No destination parameter found for #{\"price\"}. Found: #{\"id\" \"name\"}"}
                               (mt/user-http-request :crowberto :post 400 execute-path {:parameters {"id" 1 "name" "Blueberries" "price" 1234}})))))))))))
 
-(defn- ee-features-enabled? []
```

Comment:
I _hate_ commenting out code, but since we plan to come right back to this for 47 this seems like a rare case where it might make sense.

Reply:
i've 50/50 on this. Is there an issue to revisit this? If so let's toss this snippet in the issue. Otherwise leave it commented out. Just worried that whoever comes back to this will write new tests and forget about this block.